src/shared/services/map-location/index.js

```diff
@@ -46,9 +46,9 @@ export const mapLocation = loc => {
   return value;
 };
 
-const getAddressText = ({ openbare_ruimte, huisnummer, postcode, woonplaats }) =>
+const getAddressText = ({ openbare_ruimte, huisnummer, huisletter, huisnummer_toevoeging, postcode, woonplaats }) =>
   [
-    [openbare_ruimte, huisnummer],
+    [openbare_ruimte, `${huisnummer || ''}${huisletter || ''}${huisnummer_toevoeging || ''}`],
```

Comment:
Correct would be here een dash `-` between the `huisletter` and `toevoeging`. This is the standard notation and I would keep that.

Reply:
Since we're getting the full house number from the geocoder service now and since that address already has the correct notation, it's not necessary to format the address like that anymore. That's why I left the dash out.